wordpress-mobile/gutenberg-mobile
# Gallery block fix: restore focus caption prop for native

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/3510

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/32029

Alternative to https://github.com/WordPress/gutenberg/pull/31938.

To test:

1. Add a Gallery block
2. Add some images from the device or media library
3. Tap on the gallery caption
4. Verify that the editor doesn't crash

PR submission checklist:
